webrecorder/browsertrix
# Change scale of running crawls

(https://github.com/webrecorder/browsertrix-cloud/issues/143) Allows user to change crawl scale from crawl detail page, + some layout adjustments to make room for more fields and action buttons.

### Manual testing
1. Run app and start a crawl
2. Go to crawl detail page. Verify you see "Scale" option next to other action buttons
3. Click "Scale". Verify dialog opens with current scale selected
4. Choose a different scale. Verify scale updates

### Screenshots
<img width="1033" alt="Screen Shot 2022-02-25 at 12 05 34 PM" src="https://user-images.githubusercontent.com/4672952/155790999-24fdf274-1f7d-4cba-99b5-f645ef318841.png">
<img width="525" alt="Screen Shot 2022-02-25 at 12 05 37 PM" src="https://user-images.githubusercontent.com/4672952/155791025-a008523b-3498-4e7b-8d26-f8cd8d24e54b.png">
<img width="1033" alt="Screen Shot 2022-02-25 at 12 05 49 PM" src="https://user-images.githubusercontent.com/4672952/155791037-6b5c4d50-9a61-4312-a1fa-c142327a1f80.png">
